app/views/layouts/application.html.haml

```diff
@@ -24,7 +24,7 @@
           = t('app.title')
         %ul#proposition-links
           - if authenticated?
-            %li= link_to t('nav.school_page_link'), new_sessions_path
+            %li= active_link_to t('nav.school_page_link'), new_sessions_path, active: [['hiring_staff/schools'], []]
```

Comment:
This is such a small check that adding another depency seems unecessary.

We could just add a helper that returns 'active' or ''

```
def active_link_class(link_path)
    current_page?(link_path) ? 'active' : ''
end
```
### Usage
```
link_to t('nav.school_page_link'), new_sessions_path, class: active_link_class(school_path)
```

Reply:
I had no idea it could be that simple 😊